app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
The license is about https://github.com/diaspora/diaspora/pull/6095#issuecomment-111663598